app/views/users/index.html.erb

```diff
@@ -22,6 +23,7 @@
         <td><%= user.phone_number %></td>
         <td><%= user.created_at %></td>
         <td><%= user.role.humanize %></td>
+<!--        <td><%#= MembershipType.find(user&.membership_type_id).name %></td>-->
```

Comment:
I feel like here I should be able to say `user.membership_type.name` but for some reason I can't.  Is there anything obvious missing in the PR that makes this break?